Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +15,67 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        public static string Name { get; set; }
+        public static string Surname { get; set; }
+        public static int Age { get; set; }
+        public static int Weight { get; set; }
+        public static float Height { get; set; }
+
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            if (!int.TryParse(Console.ReadLine(), out int age))
+            {
+                //Console.WriteLine("Age Should be a Number");
```

Comment:
This is correct, not sure why you commented this out. Because of tests I reckon- which is fine. You will need this in hw4. So all good.